### ISSUE TITLE
Slimes grow in size after eating only when new food source is nearby

### ISSUE THREAD

**Issue by Trexdude** [VOREStation/VOREStation#8464]
#### Brief description of the bug
After a slime eats a monkey, that slime doesn't go onto the next stage of their growth until a new monkey is in their view, at which point they advance to the stage they need to advance. This happens even after a period of time passes between them eating a monkey and a new monkey appearing.

#### What you expected to happen
A slime eats a monkey and immediately or quickly advances to the next stage of their growth.

#### What actually happened
A slime that eats a monkey (possibly needs to be obedient beforehand) will not advance in growth until a new monkey appears in their view.

#### Steps to reproduce
- Make a baby slime, shoot with taser until obedient.
- Fully feed the slime a monkey.
- After the slime finishes, the baby slime will remain a baby until a new food source appears, at which point they will become an adult. If the procedure is followed with an adult slime, the adult slime will not split until a new food source appears.

#### Code Revision
- 4c340c5dac0a7afa8f5f219ec3ba812c5ed67033 
 

#### Anything else you may wish to add:
-

**Comment by Trexdude** [comment]
Something weird also happened where I died in a cell with three red-eye slimes and one regular slime, and the slimes only start attacking when I re-enter my corpse.

**Comment by crossexonar** [comment]
Yeah, can confirm that this is a biiig problem, it becomes even more of a pain when youre using Amber slimes to advanced stages.
Or even worse when using ruby extract, ruby extract means you HAVE to kill three monkeys just to get the slimes going.

**Comment by r4d6** [comment]
It is a problem, but at least you can decide when they split so you are ready to tase them for eating each other's face.

**Comment by crossexonar** [comment]
It honestly just takes the fun / danger out of doing xenobio, and when you get to a stage where youre ready to do that with loyalty extracts, it becomes a pain in the arse, since you have to kill, by yourself, three monkeys per slime instead of two (by this stage I'm usually using ambers to reproduce my slimes, which means I have to kill the monkeys anyway.)

**Comment by r4d6** [comment]
> It honestly just takes the fun / danger out of doing xenobio, and when you get to a stage where youre ready to do that with loyalty extracts, it becomes a pain in the arse, since you have to kill, by yourself, three monkeys per slime instead of two (by this stage I'm usually using ambers to reproduce my slimes, which means I have to kill the monkeys anyway.)

Wait a minute, when do you have to kill an additional monkey? I just put one in front of them, through the window, wait for them to grow/split, then put the monkey in my bag.

**Comment by crossexonar** [comment]
Inject blood into Ruby Slime extract, makes slimes passive to anything thats not in crit (including you when you go SSD)
Completely pacifies the slimes so that you have to kill the monkeighs yourself, I usually use poly-nitric for clean xenobio with no mess.
When you combine this with ambers, you get completely safe slime growth with no issues of outbreaks (assuming you light pinked your amber)
